api/opencl_runtime_layer.asciidoc

```diff
@@ -8198,6 +8198,10 @@ Otherwise, it returns one of the following errors:
   * {CL_INVALID_WORK_GROUP_SIZE} if _local_work_size_ is specified and is not
     consistent with the required number of sub-groups for _kernel_ in the
     program source.
+  * {CL_INVALID_WORK_GROUP_SIZE} if _global_work_size_ is not a multiple of
```

Comment:
Isn't this scenario already covered by the "{CL_INVALID_WORK_GROUP_SIZE} if the work-group size must be uniform and..." bullet two below this one?

Reply:
Arguably it is but I think it's useful to specify exact error conditions individually for at least the following reasons:
1. This makes it much easier to tie an error to something the application has done. How do you know when the "work-group size must be uniform"?
1. This is a new failure mode that can only happen in OpenCL 3.0, which I think is generally helpful to document. A 1.2 application developer can rule 3.0-only errors with a quick look at the spec. Granted, in this particular case, the buttet point you mentioned makes this moot.
1. If/when we develop a validation layer, having all error cases being documented separately makes for easier number / automatic processing.
1. This makes it easier to write negative tests.